tensorflow/compiler/tf2xla/kernels/reduction_ops.cc

```diff
@@ -84,7 +84,19 @@ REGISTER_XLA_OP(Name("Min").CompileTimeConstantInput("reduction_indices"),
 class MaxOp : public XlaReductionOp {
  public:
   explicit MaxOp(OpKernelConstruction* ctx)
-      : XlaReductionOp(ctx, ctx->input_type(0)) {}
+      : XlaReductionOp(ctx, ctx->input_type(0)) {
+    OP_REQUIRES_OK(ctx, TypeCheck(xla_reduction_type_));
+  }
+
+  Status TypeCheck(xla::PrimitiveType xla_reduction_type_) {
+    if (xla_reduction_type_ == xla::C64) {
```

Comment:
What about `xla::C128`?

Reply:
Hi @sanjoy Thanks for the review. 
Resolve this by adding for unsupported types based on https://github.com/tensorflow/tensorflow/blob/c48d2a48bbbb9036a9b86194a8ac6bd3160ade00/tensorflow/compiler/xla/literal_util.cc#L207-L250